src/banana/toy.py

```diff
@@ -112,7 +112,7 @@ def set(self):
 
     def hasFlavor(self, pid):
         """Contains a pdf for pid?"""
-        return pid in [21] + list(range(-6, 6 + 1))
+        return pid in [21, 22] + list(range(-6, 6 + 1))
```

Comment:
For reading easyness, add parenthesis around the full list (otherwise you rely on priority rules between `in` and `+` operators, that are not so universally clear)